source/guide_mautic.rst

```diff
@@ -0,0 +1,381 @@
+.. author:: Putzwasser <26040044+putzwasser@users.noreply.github.com>
+
+.. tag:: web
+.. tag:: lang-php
+.. tag:: audience-business
+.. tag:: customer-management
+.. tag:: mail
+
+.. highlight:: console
+
+.. sidebar:: Logo
+  .. image:: _static/images/mautic.svg
+      :align: center
+
+.. _mautic-installation:
+######
+Mautic
+######
+
+.. tag_list::
+
+Mautic is an open-source marketing automation platform that enables businesses to create, manage, and track personalized digital marketing campaigns across multiple channels and is very suitable to build an email newsletter.
+
+----
+
+This guide will show you how to install Mautic 4 on your Uberspace using the Composer installation method.
+
+.. important:: It is bad practice to run your Mautic along your blog on the same server.
```

Comment:
You can ditch this whole box, just use only "isabell.uber.space" as this is the standard procedure in all guides.

Reply:
Won't change. I don't see the point to simplify here. People use Mautic via a subdomain of their TLD. So in 99% of the use cases Mautic is set up to be reached via `(nl|newsletter|mautic).mysite.com`.

This needs a special configuration: suppressing some of the headers as explained in the guide below. This sets this guide apart from most of the other guides where a custom domain doesn't need special configuration.

Why would you make thing more complicated for users? The header thing took me quite a while to figure out. Ditching the subdomain from this guide certainly won't enhance the quality of the guide.

In the rare occasion some will use Mautic only by using their ubserspace subdomain they can simply ignore setting up the subdomain.

If you insist, please change it yourself.

Reply:
I get your point and it is worthwhile to reference to the header, but it is separated from the domain setup and furthermore linked to the usage of forms, you can check how I did it now in my commits: https://deploy-preview-1506--uberlab.netlify.app/guide_mautic/#suppressing-security-header-for-external-origins